sample-dapps/ethereum-wallet-explorer/src/app/api/wallet/nfts/route.ts

```diff
@@ -0,0 +1,27 @@
+import { ethers } from 'ethers';
+import { handleError, parseAndValidateAddress } from '../../utils/helpers';
+
+export async function GET(request: Request) {
+    const url = new URL(request.url);
+    const walletAddress = url.searchParams.get('walletAddress');
+    const page = parseInt(url.searchParams.get('page') || '1', 10);
```

Comment:
Are you using a URL search param for page?

Reply:
This is retrieving the page via the NFTsTable.tsx where i am keeping track of users state clicking between next/prev buttons